lib/grpc/GrpcService.ts

```diff
@@ -32,6 +32,7 @@ const createOrder = (order: Order) => {
   grpcOrder.setPrice(order.price);
   grpcOrder.setQuantity(order.quantity);
   grpcOrder.setSide(order.isBuy ? xudrpc.OrderSide.BUY : xudrpc.OrderSide.SELL);
+  grpcOrder.setExecutedAt(Date.now());
```

Comment:
We don't want to always set this to the current date, because then even standing, unexecuted orders (like those we get back from `ListOrders`) will show that they've been executed.

I'm actually thinking we should drop the `executed_at` proto field for now because it's not actually an event from the perspective of the API right now. Also it's a little complicated given that orders can be executed partially at various different times.

Reply:
Agree.